static/app/components/events/eventStatisticalDetector/eventFunctionComparisonList.tsx

```diff
@@ -95,21 +94,12 @@ function EventComparisonListInner({
 }: EventComparisonListInnerProps) {
   const organization = useOrganization();
 
-  // Make sure to memo this. Otherwise, each re-render will have
-  // a different min/max date time, causing the query to refetch.
-  const maxDateTime = useMemo(() => Date.now(), []);
-  const minDateTime = maxDateTime - 90 * DAY;
-
-  const breakpointTime = breakpoint * 1000;
-  const breakpointDateTime = new Date(breakpointTime);
-
-  const beforeTime = breakpointTime - DAY;
-  const beforeDateTime =
-    beforeTime >= minDateTime ? new Date(beforeTime) : new Date(minDateTime);
-
-  const afterTime = breakpointTime + DAY;
-  const afterDateTime =
-    afterTime <= maxDateTime ? new Date(afterTime) : new Date(maxDateTime);
+  const breakpointDateTime = new Date(breakpoint * 1000);
+  const datetime = useRelativeDateTime({
```

Comment:
I stole your logic for my chart bounds, would you mind slipping this change in there too? :pray: otherwise I'll just make use of this as well. Thanks!

Reply:
sure let me add that